app/main.js

```diff
@@ -70,15 +70,21 @@ app.on('ready', function() {
 
   // Create the browser window.
   mainWindow = new BrowserWindow({
+    backgroundColor: '#ffffff',
     x: mainWindowState.x,
     y: mainWindowState.y,
     width: mainWindowState.width,
     height: mainWindowState.height,
     minWidth: 298,
     minHeight: 502,
     titleBarStyle: 'hidden-inset',
-    title: 'Fog'
+    title: 'Fog',
+    show: false
   });
+  
+  mainWindow.once('ready-to-show', () => {
+    mainWindow.show()
+  })
```

Comment:
With the solution of `backgroundColor: '#ffffff'` at L73 the app started with grey then turn white. So I solved it by following this [tutorial](https://www.christianengvall.se/electron-white-screen-app-startup/) by adding
```
mainWindow.once('ready-to-show', () => {
   mainWindow.show()
 })
```
at L81-87